src/components/dev-hub/podcast-player/player-contols.js

```diff
@@ -0,0 +1,98 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+import Button from '~components/dev-hub/button';
+import FastForwardIcon from '~components/dev-hub/icons/audio-player/fast-forward-icon';
+import PauseIcon from '~components/dev-hub/icons/audio-player/pause-icon';
+import PlayIcon from '~components/dev-hub/icons/audio-player/play-icon';
+import RewindIcon from '~components/dev-hub/icons/audio-player/rewind-icon';
+
+import { size } from '~components/dev-hub/theme';
+
+const buttonStyles = css`
+    margin-right: 12px;
+    padding: ${size.tiny};
+    &:active {
+        filter: brightness(60%);
+    }
+`;
+
+export const ControlsContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    justify-content: center;
+`;
+
+const StyledTimeControlButton = styled(Button)`
+    ${buttonStyles},
+    &:hover {
+        svg {
+            transform: scale(1.15);
```

Comment:
Is it possible to not apply the `transform` to the 30 text? (Second `path` within). I like the idea since this is a `path`.

Reply:
I tried using first and second `path` but without results, so I left this fast variant. Also, we don’t need to fix margins and paddings for hover effect.